schema-engine/core/src/state.rs

```diff
@@ -113,7 +114,12 @@ impl EngineState {
         response_receiver.await.expect("receiver boomed")
     }
 
-    async fn with_connector_for_url<O: Send + 'static>(&self, url: String, f: ConnectorRequest<O>) -> CoreResult<O> {
+    /// TODO
```

Comment:
I will undo this. At some point ran the migrations by importing schema core. No longer needed.